api/nodes/disk_writer.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
+
+package synapse;
+
+message DiskWriterConfig {
+  string filename = 1;
```

Comment:
need to consider whether we overwrite, or append -- or if we can increment filenames so that we don't destroy data

an autoincrement flag or snthg would be awesome